src/utils/jsToAstParser.js

```diff
@@ -0,0 +1,18 @@
+import * as parser from "@babel/parser";
+
+export default function parseJscodeToAST(request) {
+  const { functionCode } = request.body;
```

Comment:
```suggestion
export default function parseJscodeToAST(functionCode) {
```

JavaScript 코드를 입력 받아 구상 구문 트리 `AST`를 반환하는 유틸 함수를 구현해주셨군요!
프로젝트 내에서 아주 유용하게 사용할 수 있을 것 같습니다!
또한 함수명도 해당 유틸 함수가 어떤 기능을 하는지 명확하게 알 수 있는 것 같아 좋은 것 같습니다! 👍🏻👍🏻

다만, 저희 프로젝트 흐름상 HTTP의 요청 body 에서 유저가 입력한 코드를 받아오는 것은 맞지만
**유틸 함수를 구현하신 맥락**에서는 매개변수를 `request 객체 body`에서 유저가 입력한 코드를 찾아 파싱 하는 것이아닌

**유저가 입력한 코드를 인자로 전달 받도록 구현**한다면 `사용성이 좋은 유틸 함수로 개선`할 수 있을 것 같아 
해당 부분을 개선해보면 어떨까합니다🤔

Reply:
네 해당 부분 반영하여 수정 완료 했습니다!👍🏻